hack/licence-detector/detector/detector.go

```diff
@@ -9,54 +9,66 @@ import (
 	"errors"
 	"fmt"
 	"io"
+	"io/ioutil"
 	"path/filepath"
 	"regexp"
 	"strings"
 	"time"
 
+	"github.com/elastic/cloud-on-k8s/hack/licence-detector/dependency"
+	"github.com/google/licenseclassifier"
 	"github.com/karrick/godirwalk"
 )
 
-var errLicenceNotFound = errors.New("failed to detect licence")
+// detectionThreshold is the minimum confidence score required from the licence classifier.
+const detectionThreshold = 0.85
```

Comment:
Just curious, was the 0.8 here: https://github.com/google/licenseclassifier/blob/master/classifier.go#L39
too loose?

Reply:
It's mainly for self-documentation here so that someone reading the code would be aware of the existence of a confidence score. Also, all of our current dependencies had scores higher than 0.85 so I thought it would be good to have a higher threshold to reduce the potential for a false positive slipping through.